examples/rossmann-store-sales-example.ipynb

```diff
@@ -68,12 +85,11 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
    "metadata": {},
    "outputs": [],
    "source": [
-    "DATA_DIR = os.environ.get('INPUT_DATA_DIR', './data')\n",
-    "OUTPUT_DATA_DIR = os.environ.get('INPUT_DATA_DIR', './data')\n",
+    "DATA_DIR = os.environ.get(\"OUTPUT_DATA_DIR\", \"./data\")\n",
```

Comment:
This probably should be INPUT_DATA_DIR, since it's also used for getting reading the dataset.

Right now the unittests are failling because of this in tests/unit/test_notebooks.py::test_rossman_example 

```
Traceback (most recent call last):
  File "/tmp/pytest-of-jenkins/pytest-20/test_rossman_example0/notebook.py", line 59, in <module>
    proc.apply(train_dataset, record_stats=True, output_path=PREPROCESS_DIR_TRAIN, shuffle=nvt.io.Shuffle.PER_WORKER, out_files_per_proc=2)
  File "/var/jenkins_home/nvtabular/nvtabular/workflow.py", line 729, in apply
    num_io_threads=num_io_threads,
  File "/var/jenkins_home/nvtabular/nvtabular/workflow.py", line 829, in build_and_process_graph
    self.exec_phase(idx, record_stats=record_stats)
  File "/var/jenkins_home/nvtabular/nvtabular/workflow.py", line 612, in exec_phase
    self._aggregated_dask_transform(transforms)
  File "/var/jenkins_home/nvtabular/nvtabular/workflow.py", line 587, in _aggregated_dask_transform
    ddf = self.get_ddf()
  File "/var/jenkins_home/nvtabular/nvtabular/workflow.py", line 575, in get_ddf
    return self.ddf.to_ddf(columns=columns, shuffle=self._shuffle_parts)
  File "/var/jenkins_home/nvtabular/nvtabular/io.py", line 809, in to_ddf
    ddf = self.engine.to_ddf(columns=columns)
  File "/var/jenkins_home/nvtabular/nvtabular/io.py", line 1060, in to_ddf
    return dask_cudf.read_csv(self.paths, chunksize=self.part_size, **self.csv_kwargs)[
  File "/opt/conda/lib/python3.7/site-packages/dask_cudf-0.15.0a0+4964.g4b6b7c0fd.dirty-py3.7.egg/dask_cudf/io/csv.py", line 19, in read_csv
    return _internal_read_csv(path=path, chunksize=chunksize, **kwargs)
  File "/opt/conda/lib/python3.7/site-packages/dask_cudf-0.15.0a0+4964.g4b6b7c0fd.dirty-py3.7.egg/dask_cudf/io/csv.py", line 59, in _internal_read_csv
    meta = dask_reader(filenames[0], **kwargs)._meta
  File "/opt/conda/lib/python3.7/site-packages/dask/dataframe/io/csv.py", line 649, in read
    **kwargs,
  File "/opt/conda/lib/python3.7/site-packages/dask/dataframe/io/csv.py", line 479, in read_pandas
    **(storage_options or {}),
  File "/opt/conda/lib/python3.7/site-packages/dask/bytes/core.py", line 125, in read_bytes
    size = fs.info(path)["size"]
  File "/opt/conda/lib/python3.7/site-packages/fsspec/implementations/local.py", line 60, in info
    out = os.stat(path, follow_symlinks=False)
FileNotFoundError: [Errno 2] No such file or directory: '/var/jenkins_home/nvtabular/data/train.csv'
```



```suggestion
    "DATA_DIR = os.environ.get(\"INPUT_DATA_DIR\", \"./data\")\n",
```

Reply:
Ok I think I'm confused then, because the preprocessing notebook uses INPUT_DATA_DIR to read the original data, but then places the preprocessed data with the extra features in OUTPUT_DATA_DIR, which is where `train.csv` should live. Unless we're not using the preprocessing notebook first in this test? When I run the preproc notebook followed by the example notebook locally, this works.